bootstrap.env

```diff
@@ -22,4 +22,7 @@ export APP_NAMESPACE_NAME=app-$UNIQUE_TEST_ID
 # export STOP_RUNNING_ENV=true
 # export CONJUR_ACCOUNT=cucumber
 # export CONJUR_LOG_LEVEL=debug
+# export CONJUR_AUTHENTICATORS=authn-k8s/${AUTHENTICATOR_ID}
+# Uncomment to run HELM tests
+# export HELM_TESTS=true
```

Comment:
Why do we differentiate HELM tests from the rest? I would expect this to be a parameter for the .sh file

Reply:
Deleted this (: